src/matchers/toHaveBeenCalledBefore/index.js

```diff
@@ -1,5 +1,7 @@
 import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
 
+import { isJestMockOrSpy } from '../../utils';
```

Comment:
I really like how you broke this out to the utils 💯

Reply:
Thanks! I usually try to keep such atomic functions reusable.